components/dropdown/docs/api.md

```diff
@@ -17,6 +17,7 @@
 | `chevron`               | `chevron`               |           | ` Boolean `                                      |                | If declared, the dropdown displays a chevron on the right. |
 | `common`                | `common`                |           | ` Boolean `                                      |                | If declared, the dropdown will be styled with the common theme. |
 | `commonWrapperClasses`  |                         | readonly  | `{ trigger: boolean; wrapper: boolean; hasFocus: boolean \| undefined; simple: boolean \| undefined; parentBorder: boolean; }` |                |                                                  |
+| `disableFocusTrap`      | `disableFocusTrap`      |           | `boolean`                                        |                | If declare, the focus trap inside of bib will be turned off. |
```

Comment:
I don't think this should be public. We never put focus in the bib in datepicker.